### PR TITLE
Dockerized the app and supported docker compose for single launch point

### PR DESCRIPTION
**Pull request by vigneshwarap**
## Description

1. Introduced dockerfile to run app as a individual container.
2. Introduced docker-compose.yaml to serve as base yaml for dev(docker-compose.dev.yaml) abd prod(docker-compose.prod.yaml).
3. Added entrypoint.sh for running seed db data queries.
4. Updated README.md and .env.example for env valiables update.
5. Updated seed-prod.ts to use env variables.

## Related Issues
#41

### Check List

Check both dev and prod application buildup and running
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.

### CODE REVIEW

**Comment by vigneshwarap** [comment]
Hi @AugustinMauroy 

Sorry to raise a another PR with same code change.
I just accidently raised PR with my another git username and mail id.

Please review it once your are back.
Thanks

**Comment by AugustinMauroy** [comment]
Hello, there have been changes on the main, you would be updating t'as branch and all will be good.

**Comment by vigneshwarap** [comment]
Hi @AugustinMauroy 
Updated the docker support from mysql to psql and also resolved merge conflicts.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:20
```
Comment (AugustinMauroy) [inline Dockerfile:1]: Why not use node alpine ?

**Pull request merged**